src/components/show-more/show-more.ts

```diff
@@ -0,0 +1,59 @@
+import { PluginObject } from 'vue';
+import Component from 'vue-class-component';
+import { Prop } from 'vue-property-decorator';
+import { FormatMode } from '../../../src/utils';
+import { ModulVue } from '../../utils/vue/vue';
+import { MButtonSkin } from '../button/button';
+import { SHOW_MORE_NAME } from '../component-names';
+import WithRender from './show-more.html?style=./show-more.scss';
+
+@WithRender
+@Component
+export class MShowMore extends ModulVue {
+    @Prop({ default: 0 })
+    nbVisible: number;
+
+    @Prop({ required: true })
+    nbTotal: number;
+
+    @Prop({ default: false })
+    loading: boolean;
+
+    get isVisible(): boolean {
+        return this.nbTotal !== undefined;
```

Comment:
une props ne peut jamais être undefined. Je vois mal dans quel cas cette condition va être a false?

Reply:
Une props est undefined si elle n'est pas spécifiée par le parent et qu'elle n'a pas de valeur par défaut.

Reply:
Oui mais vu que la props est à required: true cette condition ne devrait jamais arrivé en pratique . 

Je dis ca car si la props est initialisé a undefined elle ne sera jamais reactive ce qui est un problème -> ( voir https://github.com/vuejs/vue-class-component section undefined will not be reactive)

Donc si cette condition ne peut jamais arrivé a cause du required=true je suggère de l'enlever...

Reply:
Après discussion avec @jipigi, c'était pour avoir une rétroaction supplémentaire qu'un simple message d'erreur dans la console du navigateur, et ça nous évite de prendre une décision sur quoi afficher si on ne le spécifie pas.

Reply:
Pour ce qui est de la réactivité avec undefined, c'est vrai avec le data ce que tu dis, mais pas avec les props. Les props sont toujours réactives peu importes les valeur par défaut.

Reply:
👍 javais mal lue la définition vue-class-component, pour ce qui est de laisser la condition si @jipigi l'a dit !